src/bindings/python/docs/test_examples.md

```diff
@@ -17,11 +20,16 @@
 cd .../openvino/src/bindings/python/
 ```
 
-To run **all tests**:
+To run new Python API tests (since 2022.1 release):
```

Comment:
Small nit: have we agreed to call it "new Python API"? Please ignore if yes. :)

Reply:
I think it should be "OpenVINO Python API 2.0"

Reply:
I want to leave it as `OpenVINO Python API tests` as now it default API now. The old one is compatibility API

Reply:
discussed offline